npp.sls

```diff
@@ -5,11 +5,10 @@
     {% set PROGRAM_FILES = "%ProgramFiles%" %}
 {% endif %}
 npp:
-  {% for version in ['7.7.1', '7.7', '7.6.6', '7.6.4', '7.6.3', '7.6.2', '7.6.1', '7.5.9', '7.5.8', '7.5.7', '7.5.6', '7.5.5', '7.5.4', '7.5.3', '7.5.2', '7.5.1', '7.4.2'] %}
-  {% set maj_version = version[0] ~ '.x' %}
+  {% for version in ['7.9', '7.8.9', '7.7.1', '7.7'] %}
```

Comment:
Please do not remove the older versions. Salt still needs to know how to uninstall those.

Reply:
You might want to break this up into multiple PRs so the NPP changes aren't holding up the rest of the changes you want to make.

Reply:
I try to keep the old versions, but the the github verification process was crashed, because the old npp source became unavailable. In the new source (github) the oldest version was 7.7.

Reply:
@akoscomp sorry to have stepped on your PR. I have done it now, the travisci test have an option to be skipped.

Reply:
Thanks.